### PR TITLE
Update helmfile.yaml example in README

### PR DESCRIPTION
**Pull request by Vad1mo**
Added default comments into helmfile.yaml example

fixed some typos

Signed-off-by: Vadim Bauer <vb@8gears.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,22 +79,26 @@ helmDefaults:
...
+  # verify the chart before upgrading (only works with packaged charts not directories)
+  verify: true        # (default false)
+  # wait for k8s resources via --wait. (default false)
+  wait: true          # (default false) 
+  # time in seconds to wait for any individual Kubernetes operation (like Jobs for hooks, and waits on pod/pvc/svc/deployment readiness)
+  timeout: 600        # (default 300)
+  # performs pods restart for the resource if applicable
+  recreatePods: true  # (default false)
+  # forces resource update through delete/recreate if needed
+  force: true         # (default true)
```
Comment (mumoshu) [inline README.md:91]: force is false by default

File: README.md [hunk 2]
```diff
@@ -78,22 +79,26 @@ helmDefaults:
   # additional and global args passed to helm
   args:
     - "--set k=v"
```
Comment (mumoshu) [inline README.md:81]: Would it worth noting that the args is empty by default, or is it too obvious to be documented?

File: README.md [hunk 3]
```diff
@@ -69,7 +69,8 @@ repositories:
 
 # context: kube-context # this directive is deprecated, please consider using helmDefaults.kubeContext
 
-#default values to set for args along with dedicated keys that can be set by contributers, cli args take precedence over these
+# Default values to set for args along with dedicated keys that can be set by contributors, cli args take precedence over these. 
+# Non set values correspond to helm default flags with the similar name.
```
Comment (mumoshu) [inline README.md:73]:
```suggestion
# In other words, unset values results in no flags passed to helm.
# Also see the helm usage (helm SUBCOMMAND -h) for more info on default values when those flags are'nt provided.
```

File: README.md [hunk 4]
```diff
@@ -78,22 +79,26 @@ helmDefaults:
...
+  wait: true          # (default false) 
+  # time in seconds to wait for any individual Kubernetes operation (like Jobs for hooks, and waits on pod/pvc/svc/deployment readiness)
+  timeout: 600        # (default 300)
+  # performs pods restart for the resource if applicable
+  recreatePods: true  # (default false)
+  # forces resource update through delete/recreate if needed
+  force: true         # (default true)
   # enable TLS for request to Tiller
-  tls: true
+  tls: true           # (default true)
```
Comment (mumoshu) [inline README.md:93]: tls is false by default

File: README.md [hunk 5]
```diff
@@ -78,22 +79,26 @@ helmDefaults:
...
   # path to TLS CA certificate file (default "$HELM_HOME/ca.pem")
   tlsCACert: "path/to/ca.pem"
   # path to TLS certificate file (default "$HELM_HOME/cert.pem")
   tlsCert: "path/to/cert.pem"
   # path to TLS key file (default "$HELM_HOME/key.pem")
-  tlsKey: "path/to/key.pem"
-  # limit the maximum number of revisions saved per release. Use 0 for no limit (default 10)
-  historyMax: 10
+  tlsKey: "path/to/key.pem"   # (default "")
```
Comment (mumoshu) [inline README.md:99]: tls key defaults to `"$HELM_HOME/key.pem"` according to `helm upgrade -h`

File: README.md [hunk 6]
```diff
@@ -145,26 +150,22 @@ releases:
...
-    # forces resource update through delete/recreate if needed
-    force: true
-    # set `false` to uninstall on sync
+      - vault_secret.yaml    
+    # Override helmDefaults options for verify, wait, timeout, recreatePods and force. 
+    verify: true          # (default false)    
+    wait: true            # (default false)
+    timeout: 60           # (default 300)
+    recreatePods: true    # (default false)
+    force: true           # (default true)
```
Comment (mumoshu) [inline README.md:159]: force is false by default

File: README.md [hunk 7]
```diff
@@ -177,6 +178,7 @@ releases:
     # --kube-context to be passed to helm commands
     # CAUTION: this doesn't work as expected for `tilerless: true`.
     # See https://github.com/roboll/helmfile/issues/642
+    # (default "")
```
Comment (mumoshu) [inline README.md:181]:
```suggestion
    # (defaults to "", which means the standard kubeconfig, either ~/kubeconfig or the file pointed by $KUBECONFIG)
```

**Comment by Vad1mo** [comment]
@mumoshu could you do another review?

**Pull request merged**